src/apps/ocioconvert/main.cpp

```diff
@@ -625,6 +625,8 @@ int main(int argc, const char **argv)
             exit(1);
         }
 
+        spec.attribute("oiio:ColorSpace", outputcolorspace);
```

Comment:
There are three different conversions: 1) using `inputcolorspace` and `outputcolorspace`, 2) using a LUT file, and 3) using `inputcolorspace` and a (`display`, `view`) pair. Only the first conversion can add the `oiio:ColorSpace` metadata so, the code needs to check if `outputcolorspace` is not null.

Reply:
@lgritz is there a way with the `OpenImageIO` metadata to save the (`display`, `view`) pair?

Reply:
You can ask it to save any name/data you want. But there is not an established convention for encoding anything other than simply the name of the color space that the pixels are believed to be in at the point they are saved. (Also note that the ability to actually honor the request to save any of this depends on the specific file format being used. OpenEXR is pretty flexible with arbitrary metadata, most other file formats are not.)

Reply:
So my understanding is that at this point, the pixels aren't in a named color space that's in the config per se.

Is there an established OCIO convention for using a string to denote a (display, view) combination? And what could we do with that if it was stored in the file? Is there any OCIO API call that can use that as a starting point? Like, can you say "these pixels are values in this view of this display, now transform it into some other space?"

Reply:
There is no OCIO approved convention (but Maya has one). 
That's a good point to bring to the next OCIO TSC meeting (I will add it to the agenda)

Reply:
I would love if OCIO codified such a convention -- out of curiosity, what convention does Maya use? 

Related, I think it would be pretty useful if we had Display and View FormatMetadata enums, to complement the existing CLF InputDescriptor and OutputDescriptor enums. (Granted, "OutputDescriptor" can be anything; but it would be great to have a semi-formal convention for differentiating between color space conversion and display-view transforms, such that a system with a-priori knowledge of Display descriptors -- such as an OCIO config, for example -- would ostensibly be able to programmatically isolate the "Display" portions of the transform from the "View" portions, if that makes sense)

Reply:
One of the many motivations for the changes to the Display/View system in OCIO v2 was to have a more generic display color space available for this sort of tagging.  So if it used a v2 style Display/View, you don't care about the view and would just write the display color space.  If it's a v1 style Display/View, then by definition there is already a color space in the config that implements it and the name of that could be written.  However, since that color space would incorporate the view transform, it may not be that meaningful.